README.md

```diff
@@ -1 +1,109 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+JSON is a widely use data format. When it comes to Node applications, developers like to use JSON file to store various data such as:
+
+* Configurations
+* Seed data (initial data to populate the development or test database)
+* Settings
+* Plugins for frameworks
+
+Let's implement code which reads a JSON file and parses the file data into JavaScript object (`read.js`). The `customers.json` has information about customers including their emails, balance, phone numbers and addresses. Please don't share this information with anyone.
+
+```js
+var fs = require('fs')
+fs.readFile('./customers.json', {encoding: 'utf8'}, function(error, jsonString){
+  try {
+    var data = JSON.parse(jsonString)
+  } catch(e) {
+    console.log(e)
+  }
+  console.log(data.length, data[2].email)
```

Comment:
If we hit the `catch` block above, this line will still execute and then throw errors. Better to `return` or `throw` above, right?